### PR TITLE
Update object-detection to thresholded metrics

### PR DESCRIPTION
**Pull request by ly-kolena**
### Please check if the PR fulfills these requirements

- [ ] Include reference to internal ticket and/or GitHub issue "Fixes #NNNN" (if applicable)
- [ ] Relevant tests for the changes have been added
- [ ] Relevant docs have been added / updated

### CODE REVIEW

File: examples/dataset/object_detection_3d/object_detection_3d/utils.py [hunk 1]
```diff
@@ -181,3 +184,65 @@ def load_data(df_raw: pd.DataFrame) -> pd.DataFrame:
         )
 
     return pd.DataFrame(records)
+
+
+def _compute_thresholded_metrics(
```
Comment (bagalaster) [inline examples/dataset/object_detection_3d/object_detection_3d/utils.py:189]: What's the reason for re-implementing these evaluation functions here? Could we leverage `multiclass_datapoint_metrics` from `kolena/_experimental/object_detection/dataset.py` instead?

Comment (ly-kolena) [reply examples/dataset/object_detection_3d/object_detection_3d/utils.py:189]: The underlying data structures are not the same. for object-detection 2d, we're using our own implementation and `MatchInfereces` etc. classes, but for object-detection 3d, we're using mmdetection3d.

**Review by munkyshi — APPROVED** [review]
Read through the code a couple of times -- this all seems reasonable to me. I'd like to run through an example with this tomorrow, but I don't want to block merging this PR on carving out the time to do that.

**Comment by jared-jewitt** [comment]
Should we be updating this section on the docs? https://docs.kolena.com/dataset/formatting-your-datasets/#formatting-results-for-object-detection

**Comment by ly-kolena** [comment]
> Should we be updating this section on the docs? https://docs.kolena.com/dataset/formatting-your-datasets/#formatting-results-for-object-detection

Any objection to follow up in a separate PR? We can discuss how to present this feature and where to put there.

**Pull request merged**